### DISCUSSION TITLE
Best place to discuss automation basics?

### DISCUSSION THREAD

**Discussion by GrahamLea** [amberelectric/public-api#98] [General]
I'm keen to eventually use the Amber API to automate a bunch of appliances based on a combo of my solar output and the grid price. Right now though, I'm still at the stage of designing a solar system / selecting products, and need to get help on what products available in Australia are best suited for setting this kind of thing up.

What are the best places online for me to learn about, and/or ask newbie questions about, that kind of stuff?

**Comment by madpilot** [comment]
Such a good question.

Perhaps we should setup a community list?

**Reply by scotta1725** [reply]
Not specific to Solar but for general Home Automation questions the face book group Home Automation Australia is not a bad start. However, results will vary as there is a high level mix of experience and skills. ;-).

FYI - I use the API to turn off my Aircons and other high usage devices if a peak is hit. As well as use madpilot's template to display price (colour) on an ESPHome LED light.

**Comment by GrahamLea** [comment]
Good suggestion from Scott above leads to this FB group: https://www.facebook.com/groups/homeautoaus/about
